misc/book/convert_to_md_key.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep  28 20:01:12 2018
+
+@author: aleksandra
+"""
+import argparse
+from convert_to_md_function import convert
+
+parser = argparse.ArgumentParser()
+parser.add_argument('file_name', help = 'Name of a python script you want to convert to Markdown. Example: my_file.py', type=str)

```

Comment:
It can be useful to take multiple `file_name` arguments and run `convert()` for all of them

Reply:
Can't figure out what name would be ok, since I see no difference. Explain, please